.changeset/thirty-files-live.md

```diff
@@ -0,0 +1,21 @@
+---
+"@layerzerolabs/test-setup-devtools-evm-hardhat": minor
+"@layerzerolabs/ua-devtools-evm-hardhat-v1-test": minor
+"@layerzerolabs/test-devtools-evm-hardhat": minor
+"@layerzerolabs/ua-devtools-evm-hardhat-test": minor
+"@layerzerolabs/ua-devtools-evm-hardhat": minor
+"@layerzerolabs/devtools-evm-hardhat-test": minor
+"@layerzerolabs/devtools-evm-hardhat": minor
+"@layerzerolabs/export-deployments-test": minor
+"@layerzerolabs/toolbox-hardhat": minor
+"@layerzerolabs/devtools-cli-test": minor
+"@layerzerolabs/devtools-evm-test": minor
+"@layerzerolabs/oft-adapter-example": minor
+"@layerzerolabs/oft-solana-example": minor
+"@layerzerolabs/test-evm-node": minor
+"@layerzerolabs/onft721-example": minor
+"@layerzerolabs/oapp-example": minor
+"@layerzerolabs/oft-example": minor
+---
+
+Update hardhat runtime environment factory to match the new environment constuctor
```

Comment:
Also super small nit: there's a typo in `constructor`

Reply:
Oke will fix in another PR - we actually have one PR that adds spellcheck to CI/CD